pkgs/development/haskell-modules/make-package-set.nix

```diff
@@ -230,7 +230,7 @@ in package-set { inherit pkgs stdenv callPackage; } self // {
     # build that package.
     developPackage =
       { root
-      , name ? builtins.baseNameOf root
+      , name ? if builtins.typeOf root == "path" then builtins.baseNameOf root else ""
```

Comment:
When would you want this?  `root` gets passed to `callCabal2nix`, so it looks like it has to be a path?

What bug are you trying to fix here?

Reply:
Ah, email response doesn't work for review comments. My response is below